4-layers/Dockerfile.better

```diff
@@ -0,0 +1,25 @@
+FROM ruby:alpine
+
+RUN apk update && \
```

Comment:
Ways to make this even better?

1. Investigate the difference between `--no-cache` and `rm -rf /var/cache/apk/*` - do you need both?
2. Investigate the difference between `apk update` and `apk -U upgrade`, are they both needed, are they in the right place?